### ISSUE TITLE
weird behavior?  Can't find symbol in typechecking environment, even after restarting cli.

### ISSUE THREAD

**Issue by aryairani** [unisonweb/unison#348]
```
[info] Running org.unisonweb.BootstrapStream 42441
master> branch

    either
  * master

master> list Either

  

master> branch either
either> list Either

  type Either
  Either.leftToOptional  : Either a b -> Optional a
  Either.>>=             : Either a b -> (b -> Either a c) -> Either a c
  Either.fromRight       : b -> Either a b -> b
  Either.either          : (a -> c) -> (b -> c) -> Either a b -> c
  Either.fromLeft        : a -> Either a b -> a
  Either.isRight         : Either a b -> Boolean
  Either.rightToOptional : Either a b -> Optional b
  Either.isLeft          : Either a b -> Boolean
  Either.Left            : a -> Either a b
  Either.Right           : b -> Either a b

either> 
I don't know about the type Either.  Make sure it's imported and spelled correctly:

    1 | > (Either.Left 3 : Either Nat Nat)

either> 
I'm not sure what Either.Left means at line 1, columns 3-14

    1 | > Either.Left 3

Whatever it is, it has a type that conforms to Nat -> o.

either> 
```
I think this is distinct from #347, because restarting the CLI doesn't help.

Pretty much at a loss.

We could probably use some CLI debug commands: print the namespace, print the typechecking environment, and what else?

**Comment by aryairani** [comment]
[codebase.zip](https://github.com/unisonweb/unison/files/2879921/codebase.zip)

**Comment by aryairani** [comment]
Seems fixed as of 3aee1737.
